docs/api.md

```diff
@@ -1067,27 +1153,249 @@ Example Response Body:
+
+Query Params:
+| Field Name     | Description                                                     |
+|----------------|-----------------------------------------------------------------|
+| isLiquidatable | If set to true, returns accounts that are yet to be liquidated  |
+
+Example Response Body:
+
+```json
+{
+  "accounts": [
+    {
+      "owner": "0x000f7f22bfc28d940d4b68e13213ab17cf107790",
+      "market": "PBTC-USDC",
+      "orderNumber": "1011196300330012",
+      "cachedMargin": "36759577102",
+      "margin": "36759577102",
+      "position": "1201838526",
+      "cachedIndexValue": "1.211642619304704688",
+      "indexTimestamp": "1590100515"
+    },
+    {
+      "owner": "0x003f480be5b68c5e863d2990d043f3b58f64473f",
+      "market": "PBTC-USDC",
+      "orderNumber": "1010984800560005",
+      "cachedMargin": "368456014",
+      "margin": "368468282",
+      "position": "-3000000",
+      "cachedIndexValue": "1.207553221962107213",
+      "indexTimestamp": "1590072186"
+    }
+  ]
+}
+```
+
+#### Account Response Object
+
+| Field Name      | Description                                                                    |
+|-----------------|--------------------------------------------------------------------------------|
+| owner           | The user's wallet address.                                                     |
+| balances        | An object with the user's balances for each market.                            |
+| margin          | This is calculated as `cachedMargin- (indexValue - globalIndexValue)*position` |
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is the last stored margin value                                           |
+| pendingMargin   | This is the (pending) component of the margin when a fill is still pending     |
+| pendingPosition | This is the (pending) component of the position when a fill is still pending   |
+| indexValue      | The value of the global index from the last interaction with the account       |
+| indexTimestamp  | The timestamp when the index value was set                                     |
+
+
+# Solo and Perpetual endpoints
+
+Description:
+Gets the perpetual and solo standard actions for a particular user.
+
+Query Params:
+
+| Field Name | Description                                                     |
+|------------|-----------------------------------------------------------------|
+| owner      | The wallet address of the user.                                 |
+| type       | The type of standard action e.g. `OPEN`, `CLOSE`, `DEPOSIT`.    |
+| market     | The market of the action e.g. `PBTC-USDC`.                      |
+| limit      | The maximum number of standard actions to retrieve.             |
+| product    | The product of the standard action, e.g. `perpetual` or `solo`. |
+
+Standard Action types:
+
+| Type | Product |
+|------|---------|
+|DEPOSIT | solo, perpetual |
```

Comment:
You might already be using something but just in case it's helpful, I found this auto-formatter useful: https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter